dapp-example/tests/support/bootstrap.js

```diff
@@ -0,0 +1,110 @@
+const HederaSDK = require('@hashgraph/sdk');
+const ethers = require('ethers');
+const hethers = require('@hashgraph/hethers');
+const fs = require('fs');
+const path = require('path');
+
+const client = HederaSDK.Client
+  .forNetwork(JSON.parse(process.env.HEDERA_NETWORK))
+  .setOperator(process.env.OPERATOR_ID_MAIN, process.env.OPERATOR_KEY_MAIN);
+
+const createAccountFromCompressedPublicKey = async function(compressedPublicKey) {
+  const transferTransaction = await (new HederaSDK.TransferTransaction()
+    .addHbarTransfer(HederaSDK.PublicKey.fromString(compressedPublicKey).toAccountId(0, 0), new HederaSDK.Hbar(1000))
+    .addHbarTransfer(HederaSDK.AccountId.fromString(process.env.OPERATOR_ID_MAIN), new HederaSDK.Hbar(-1000))
+    .setNodeAccountIds([HederaSDK.AccountId.fromString('0.0.3')]))
+    .execute(client);
+
+  const txTransaction = await (new HederaSDK.TransactionRecordQuery()
+    .setTransactionId(transferTransaction.transactionId))
+    .execute(client);
+
+  const transfer = txTransaction.transfers.find(trans => !trans.amount.isNegative() && trans.accountId.num.toNumber() > 999);
+  const accountId = transfer.accountId.toString();
+
+  console.log(`Account has been successfully created: ${accountId}`);
+
+  return { accountId };
+};
+
+const createHTSToken = async function() {
+  const expiration = new Date();
+  expiration.setDate(expiration.getDate() + 30);
+  const tokenCreate = await (await new HederaSDK.TokenCreateTransaction()
+    .setTokenName('HTS_Token')
```

Comment:
Please randomize the name and symbol similar to acceptance tests.
We don't want any conflicts where run against deployed envs